fabric/path.go

```diff
@@ -0,0 +1,43 @@
+package fabric
+
+import (
+	"regexp"
+
+	"github.com/zalando/skipper/eskip"
+	"github.com/zalando/skipper/predicates"
+)
+
+var (
+	slash        = `/`
+	star         = `\*`
+	curlyOpen    = `{`
+	curlyClose   = `}`
+	end          = `$`
+	wildcardName = `(?P<name>[\w-]+?)`
+	repls        = []struct {
+		match *regexp.Regexp
+		repl  string
+	}{
+		{regexp.MustCompile(slash + star + wildcardName + slash), `/:$name/`},
+		{regexp.MustCompile(slash + star + wildcardName + end), `/:$name`},
+		{regexp.MustCompile(curlyOpen + wildcardName + curlyClose), `:$name`},
+		{regexp.MustCompile(slash + star + slash), `/:id/`},
+		{regexp.MustCompile(slash + star + end), `/:id`},
+	}
+)
+
+// fabricPathToPredicate takes a Fabric path string and transforms it an equivalent Skipper Path* predicate
+func fabricPathToPredicate(fp string) *eskip.Predicate {
+	if fp == "/**" {
```

Comment:
What about `/foo/**` or this case does not exist?

Reply:
In original version (Scala one) only `/**` is represented as `PathSubtree`. Rest just lands to `Path`. I do not see a reason why just `ends with /**` is not also transformed to `PathSubtree`. Maybe we can ask original Fabric guys.

Reply:
Asked in our Fabric and Skipper channel.